### DISCUSSION TITLE
Do you have a performance checker for FaskAPI? The problem is that when you make 100 concurrent requests, the API returns are slow. How do you solve it?

### DISCUSSION THREAD

**Discussion by lishaozheng** [fastapi/fastapi#12925] [Questions] [answered]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

![image](https://github.com/user-attachments/assets/531ebf3c-0d76-4241-a742-b9286e20736d)
![image](https://github.com/user-attachments/assets/7942db47-2a25-4af7-aaa1-4a6c40ccd735)
![image](https://github.com/user-attachments/assets/c91ae68c-e57c-471b-a182-b0a49dea3df6)
Is there a problem with the code, is there any code to refer to, or is there a health monitoring tool to check the FaskAPI

**Comment by YGuang233** [comment, marked as answer]
1. 你核心的代码有没有包含耗时的io操作?或其他的耗时操作，调用了第三方的API。
2. 检查你代码中的`await`有没有添加上
3. 使用uvicorn为服务器运行fastapi进行并发测试的时候，你可以设置workers=4，或者别的数字，但不要是1
workers用于指定启动的并发工作进程数量
但是你没有贴出你的代码？我感觉这明显和fastapi无关
![image](https://github.com/user-attachments/assets/5a52ef30-a168-46e1-9a0a-f8107306fa9f)
![image](https://github.com/user-attachments/assets/603da180-1cf3-489b-8116-68c54abd66aa)
我使用postman进行了两次测试，一次100的并发，一次3的并发连续100次请求
请求的内容是模拟用户登陆,生成jwt并返回用户信息
![image](https://github.com/user-attachments/assets/cd6b14b1-1a1f-4121-bcaf-48e6fc9e8cc4)

**Reply by lishaozheng** [reply]
谢谢
